calendars/workshops.yaml

```
@@ -164,3 +164,14 @@ events:
     repeat:
       interval: {days: 1}
       until: 2024-09-19 14:00:00
+
+  # CR train the trainer 2024 workshop 
+  - &cr-ttt-2024
+    summary: Preliminary reservation: CodeRefinery train the trainer workshop, 2024
```

```suggestion
    summary: "Preliminary reservation: CodeRefinery train the trainer workshop, 2024"
```
This neds quotation, since you have a colon in the line. Otherwise yaml thinks this colon is a mapper.